metaflow/plugins/cards/ui/src/main.ts

```diff
@@ -5,9 +5,13 @@ let app;
 
 // wrapping in try/catch to let user know if its missing #app
 try {
+  const cardDataId: string = (window as any).mfCardDataId as string;
+  const containerId: string = (window as any).mfContainerId as string;
+  const containedApp = document.querySelector(`[data-container="${containerId}"]`)?.querySelector(".card_app") as Element
```

Comment:
can these selectors be chained like: `querySelector([data-container="${containerId}"] .card_app)`?  or no.

Reply:
It may not exist, as in the case where we run `yarn dev` so I have to check it first.

and...you really made this comment so I would notice the console.log below that I need to remove!